lms/static/js/instructor_dashboard/ecommerce.js

```diff
@@ -17,12 +17,18 @@ var edx = edx || {};
         },
         clicked: function (event) {
             if (event.currentTarget.checked) {
-                this.$el.find('#coupon_expiration_date').show();
+                this.show(this.$('#coupon_expiration_date'));
                 this.$el.find('#coupon_expiration_date').focus();
             }
             else {
-                this.$el.find('#coupon_expiration_date').hide();
+                this.hide(this.$('#coupon_expiration_date'));
             }
+        },
+        show: function ($el) {
```

Comment:
The behavior of `$.fn.show()` and `$.fn.hide()` changed such that jasmine tests failed expecting certain CSS to be present. I implemented `show` and `hide` helper functions in order to preserve the old behavior.